### PR TITLE
Created a JavaScript function called zipSelectInDates that queries an…

### PR DESCRIPTION
**Pull request by DCH-CH**
…d sends geo coordinates based on dates requested by the user

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "server:dev": "nodemon index.js",
     "test": "jest --coverage",
     "test:watch": "jest --watch",
-    "lint": "eslint --fix '**/*.js'",
+    "lint": "eslint --fix \"**/*.js\"",
```
Comment (DCH-CH) [inline server/package.json:10]: For some reason lint was complaining about the ' ' quotes, so I replaced them with escaped double quotes

**Review by DCH-CH — COMMENTED** [review]
Just added some changes

File: server/package.json (hunk 1 above)
Comment (macho-catt) [reply server/package.json:10]: Yeah I think this is a weird linux / windows thing. I tested this part with your updates and it still works, so this part is good!

**Review by macho-catt — CHANGES_REQUESTED** [review]
Hi @DCH-CH 

So from my initial testing, this looks good. However, for the sake of clarity and making our endpoints more succinct, I wonder if there's a way for you to combine drawSelect and drawSelectInDates as one endpoint, so that the startDate and endDate parameters are optional. So if you make an api request to the endpoint for drawSelect, you can do it either with just the polygon parameter, or with the polygon parameter and both startDate and endDate parameters.

Could you also combine zipSelect and zipSelectInDates endpoints, as well as zipGraphs and zipGraphsInDates endpoints.

Let me know if that makes sense or you need more clarification. Here's a possible source from [stackoverflow](https://stackoverflow.com/questions/41736413/multiple-optional-route-parameters-in-express).

Additionally, can you write tests / modify the current tests for the new endpoints, and make sure they succeed? There should already be tests here that you can follow: server/__tests__/controllers.test.js

LMK if you have any questions. Thanks.

**Comment by macho-catt** [comment]
Accidentally closed it, reopening. Sorry!

**Pull request closed without merging**